vale/styles/jargonLint/Technology.yml

```diff
@@ -6,256 +6,361 @@ ignorecase: true
 level: warning
 message: Use '%s' instead of '%s'
 swap:
+  # Disable yamlint rule about line length
+  # yamllint disable-line rule:line-length
+  #
+  # Sources
+  # - https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.jsonc
+  # - https://github.com/paradite/frontend-encyclopedia/blob/main/scripts/data.json
+  # - https://github.com/huacnlee/autocorrect/blob/main/.autocorrectrc.template
+  # - many others
+  #
+  # Restore yamlint rule about length
+  # yamllint enable rule:line-length
+  #
+  # Ad hoc: Ad hoc # risk of false positives
   ACL: ACL
   ACLs: ACLs
+  ActionScript: ActionScript
   ActiveMQ: ActiveMQ
   Adwords: Adwords
   AirPods: AirPods
   AJAX: AJAX
   Aliyun: Aliyun
   Alpine: Alpine
   Altassian: Altassian
-  Angular: Angular
-  Ansible: Ansible
-  Apache: Apache
+  Amazon S3: Amazon S3
+  Android: Android
+  Angular ?JS: AngularJS
+  Ansible(?!\.com): Ansible
+  Apache(?!\.org): Apache
   API: API
   Apify: Apify
   APIs: APIs
+  App[ ]?Store: App Store
   AppKit: AppKit
   AppleScript: AppleScript
-  App[ ]?Store: App Store
+  ASCII: ASCII
+  (?<!\.)AsciiDoc: AsciiDoc  # also a file extension
+  AST: AST
   AWS: AWS
   Bitbucket: Bitbucket
   BitTorrent: BitTorrent
-  Browserify: Browserify
-  BrowserStack: BrowserStack
+  Browserify(?!\.org): Browserify
+  BrowserStack(?!\.com): BrowserStack
   CacheStorage: CacheStorage
+  CalDAV: CalDAV
+  CC[ -]?BY[ -]?4(?:\.0)?: CC BY 4.0 # License
   CDN: CDN
-  CentOS: CentOS
-  CircleCI: CircleCI
+  CentOS(?!\.org): CentOS
+  ChatGPT: ChatGPT
+  CircleCI(?!\.com): CircleCI
   CLI: CLI
-  Cloudflare: Cloudflare
+  Cloudflare(?!\.com): Cloudflare
   CloudFront: CloudFront
-  CoffeeScript: CoffeeScript
+  CMake: CMake
+  CoffeeScript(?!\.org): CoffeeScript
   Common[ .]?js: CommonJS
-  Consul: Consul
+  CommonMark(?!\.org): CommonMark
+  Consul(?!\.io): Consul
   CORS: CORS
   CPU: CPU
   CPUs: CPUs
   (?<!\.)CSON: CSON  # also a file extension
   (?<!\.)CSV: CSV  # also a file extension
-  Cypress: Cypress
-  D3.js: D3.js
+  cURL: cURL
```

Comment:
I'm unsure about this one. The preference seems to be ``curl`` from the [main website](https://curl.se/). While the [GitHub repository](https://github.com/curl/curl?tab=readme-ov-file) seems like it doesn't care too much since there they also wrote ``Curl``.

As such, I suggest that we either suggest ``curl`` or don't add this to the rule at all.

Reply:
you are right. I'm unsure where I found this one.

Also thanks for quoting the curl page, they have tons of protocols name